src/poliastro/tests/tests_twobody/test_angles.py

```diff
@@ -240,3 +241,34 @@ def test_convert_coe_and_rv_circular_equatorial(circular_equatorial):
     k, expected_res = circular_equatorial
     res = rv2coe(k, *coe2rv(k, *expected_res))
     assert_allclose(res, expected_res, atol=1e-8)
+
+
+def test_raan_from_ltan_metopb():
+    # MetOp-B: LTAN: 21:31:45
+    # METOP-B
+    # 1 38771U 12049A   20049.95408566 -.00000014  00000-0  13607-4 0  9997
+    # 2 38771  98.7092 110.9899 0001263  48.5458 295.8781 14.21485930385043
+
+    ltan = (21 + ((31 + 45 / 60) / 60)) * u.hourangle
+    epoch = astropy.time.Time(
+        astropy.time.Time("2020-01-01 00:00").to_value("mjd") + 49.95408566 - 1,
+        format="mjd",
+    )
+    expected_raan = 110.9899 * u.deg
+    raan = angles.raan_from_ltan(epoch, ltan)
+    assert_allclose(raan.wrap_at(360 * u.deg).to(u.deg), expected_raan, atol=0.3)
+
+
+def test_raan_from_ltan_sentinel5p():
+    # SENTINEL-5P
```

Comment:
I guess we could have a similar comment like

```
# LTAN: 13:30:00
```

with the source included?

Reply:
Added comment stating LTAN & source